Example/Podfile

```diff
@@ -3,8 +3,8 @@ source 'https://github.com/CocoaPods/Specs.git'
 use_frameworks!
 
 target 'RxCoreData_Example' do
-  pod 'RxSwift', '~> 4.0'
```

Comment:
Can you explain the version bump here?

Reply:
CocoaPods will not be able to resolve `RxSwift` dependency, as `Podfile` requires version `~> 4.0.0`, while `RxCoreData.podspec` specifies  `~> 4.2.0`